lib/components/LazyImage.vue

```diff
@@ -60,6 +60,13 @@ export default {
       default () {
         return true
       }
+    },
+
+    caption: {
```

Comment:
Wollen wir caption wirklich in LazyImage verwalten?

Reply:
Ne kommt raus, Slot abfrage reicht, wäre hier ja Optional.